packages/atlas/src/components/NftTile/NftTile.tsx

```diff
@@ -1,23 +1,23 @@
 import React, { useMemo, useState } from 'react'
 
-import { NftTileDetails } from '@/components/NftTileDetails'
 import { Pill, PillGroup } from '@/components/Pill'
 import { SvgActionAuction, SvgActionBuyNow, SvgActionNotForSale, SvgActionShow } from '@/components/_icons'
 import { VideoThumbnail, VideoThumbnailProps } from '@/components/_video/VideoThumbnail'
 import { formatNumberShort } from '@/utils/number'
 import { formatDurationShort } from '@/utils/time'
 
 import { Container } from './NftTile.styles'
+import { NftTileDetails } from './NftTileDetails'
 
 export type Member = {
   assetUrl?: string | null
-  name: string
+  name?: string
   onClick?: () => void
   loading?: boolean
 }
 
 export type NftTileProps = {
-  auction?: 'none' | 'minBid' | 'topBid' | 'waiting'
+  nftState?: 'idle' | 'on-sale' | 'auction'
```

Comment:
maybe we could rename this - `nftStatus` or `nftAuctionStatus` or `nftSaleStatus`? Wdyt?

Reply:
I called it nftState on the NFT widget as well but we would def benefit from having this type defined somewhere to unify NFT logic a bit as everyone seems to be doing it differently.

Reply:
nftAuctionStatus or nftSaleStatus I think wouldn't cut it because those are subtypes within the type